addons/account/static/src/js/reconciliation/reconciliation_action.js

```diff
@@ -88,8 +88,10 @@ var StatementAction = AbstractAction.extend({
         var self = this;
         var def = this.model.load(this.params.context).then(this._super.bind(this));
         return def.then(function () {
-                var title = self.model.bank_statement_id  && self.model.bank_statement_id.display_name;
-                self._setTitle(title);
+                var title = (self.model.statement && self.model.statement.statement_name) || 
```

Comment:
var title = self.model.bank_statement_id ? self.model.bank_statement_id.display_name : self.title;

Reply:
Like in 12.0